### PR TITLE
burn this stuff

### PR DESCRIPTION
**Pull request by agend**

### CODE REVIEW

File: src/bin/bobp.rs [hunk 1]
```diff
@@ -1,3 +1,136 @@
...
+    }
+}
+
+fn help() {
+    println!("usage: bobp <target ip> <target port> <payload size> <worker count> <record count>");
+}
+
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
```
Comment (pyakushin) [inline src/bin/bobp.rs]: I think, it is more convenient to use clap.rs for args parsing

File: src/bin/bobp.rs [hunk 2]
```diff
@@ -1,3 +1,136 @@
...
+            high_idx: task_conf.low_idx + task_size * (i+1),
+            payload_size: task_conf.payload_size
+        };
+        workers.push(thread::spawn(move || {    
+            bench_worker(nc, tc, stat_inner);
+        }));
+    }
+
+    for worker in workers {
+        let _ = worker.join();
```
Comment (pyakushin) [inline src/bin/bobp.rs]: I think, it's better to use `.join().unwrap()`, or match result

File: src/bin/bobp.rs [hunk 3]
```diff
@@ -1,3 +1,136 @@
...
+            low_idx: task_conf.low_idx + task_size * i,
+            high_idx: task_conf.low_idx + task_size * (i+1),
+            payload_size: task_conf.payload_size
+        };
+        workers.push(thread::spawn(move || {    
+            bench_worker(nc, tc, stat_inner);
+        }));
+    }
+
+    for worker in workers {
```
Comment (pyakushin) [inline src/bin/bobp.rs]: Another way: workers.drain(..).for_each(|w| w.join().unwrap());

**Comment by ikopylov** [comment]
LGTM

**Pull request merged**